code/src/index.css

```diff
@@ -11,3 +11,128 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+header {
+  background-color: #ffadad; 
+  border-bottom: 5px dotted;
+}
+header h1 {
+  font-family: "Courier New",
+    monospace;
+  margin: 0;
+  padding: 50px;
+  text-align: center;
+}
+footer {
+  background-color: #ffadad; 
+  border-top: 5px dotted;
+  font-family: "Courier New",
+    monospace;
+
+  padding: 30px;
+  text-align: center;
+}
+
+p {
+font-family: "Courier New",
+    monospace;
+    margin: 10px;
+}
+
+.newThought {
+  background-color: #f2f0f0;
+  box-shadow: 8px 8px black;
+}
+
+input {
+  width: 80%;
+  height: 30px;
+  margin: 8px;
+}
+.container {
+display:flex;
+flex-direction: column;
+min-width: 334px;
+max-width: 500px;
+margin: 0 auto;
+}
+
+.form-container {
+  margin: 10px;
+}
+.list {
+  width: 80%;
+  margin: 0 auto;
+  border:1px solid black;
+  margin-top: 30px;
+  box-shadow: 8px 8px black;
+}
+
+.grid {
+  display:grid;
+  Grid-template-columns: 1fr 1fr;
+}
+.message {
+  grid-column: span 2;
+}
+
+.loading-overlay {
+  position:absolute;
+  top: 0;
+  right: 0;
+  bottom:0;
+  left:0;
+  background: #c4c1c1;
+  display:flex;
+  justify-content: center;
+  align-items: center;
+}
+.loading-spinner{
```

Comment:
I loved that you did a special spinner. If you want to try it out in the future there are a lot of spinners available as packages to download! It was very easy to implement =) I will try to do my own next time to actually understand keyframes!